Add getCensusWeight handler

NateWilliams2 opened the pull request:
Adds getCensusWeight handler, tests as well

This handler uses the arbo tree `IterateLeaves()` method. @arnaucube performance-wise, is this method realistic to use with large census sizes? 

Closes #463

NateWilliams2 committed:
Add getCensusWeight handler

arnaucube reviewed (commented):
Yes, as you comment @NateWilliams2 , this can be a bit intensive from performance point of view. It will walk over all the nodes & leaves of the tree, so if the tree is big enough, it may take more than 10 seconds for example, even more than 1 minute.
Maybe we can go with this for now, if the `getCensusWeight` is not going to be used too much, having all this in mind.
Maybe another option would be to add the functionality of storing the total census weight in the `censustree` package at `Add` & `AddBatch` methods (https://github.com/vocdoni/vocdoni-node/blob/master/censustree/censustree.go#L153).
In this way, the total weight is updated in the key-value database each time that a leaf is added to the census, and when somebody calls `getCensusWeight` internally it just needs to get the 'total census weight' from the keyvalue db, instead of iterating over all the nodes and leaves of the tree.

NateWilliams2 committed:
Store censusWeight in tree database

arnaucube committed:
Add store census weight on AddBatch & ImportDump

- Updated censustree.Add to use the same db.WriteTx for the censusWeight
  & leaf addition (for atomicity)
- Added store censusWeight on AddBatch & ImportDump
- Added unit tests for censustree.GetCensusWeight method in different
  use cases
